frontend/src/pages/signup/hooks/usePasswordInput.ts

```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react';
+
+import { validateLength } from '../../../common/utils/validateLength';
+import { ERROR_MESSAGE } from '../constants/errorMessage';
+import { PASSWORD } from '../constants/password';
+
+const usePasswordInput = () => {
+  const [password, setPassword] = useState('');
+  const [passwordConfirm, setPasswordConfirm] = useState('');
+
+  const handlePasswordChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword(e.target.value);
+  };
+
+  const handlePasswordConfirmChange = (
+    e: React.ChangeEvent<HTMLInputElement>,
+  ) => {
+    setPasswordConfirm(e.target.value);
+  };
+
+  const getPasswordErrorMessage = () => {
+    if (password.length === 0) {
+      return '';
+    }
+
+    if (
+      !validateLength({
+        min: PASSWORD.MIN_LENGTH,
+        max: PASSWORD.MAX_LENGTH,
+        value: password,
+      })
+    ) {
+      return ERROR_MESSAGE.INVALID_PASSWORD_LENGTH;
+    }
+
+    return '';
+  };
+
+  const getPasswordConfirmError = () => {
+    if (password.length === 0 || passwordConfirm.length === 0) {
+      return '';
+    }
+
+    if (password !== passwordConfirm) {
+      return ERROR_MESSAGE.NOT_MATCH;
+    }
+
+    return '';
+  };
+
+  const passwordErrorMessage = getPasswordErrorMessage();
+  const passwordConfirmErrorMessage = getPasswordConfirmError();
+
+  return {
+    password,
+    passwordConfirm,
+    handlePasswordChange,
+    handlePasswordConfirmChange,
+    passwordErrorMessage,
+    passwordConfirmErrorMessage: getPasswordConfirmError(),
```

Comment:
p4: 위에 변수를 만들어놔서 이렇게 수정해도 될 거 같아 👍
```suggestion
    passwordConfirmErrorMessage
```

Reply:
수정커밋: [refactor: 만들어둔 상수화 사용하도록 수정](https://github.com/woowacourse-teams/2025-Fit-toring/pull/245/commits/68fad6d414d15b714b41232b865de6806b10580f)